extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -660,6 +666,16 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
     return (Date.now() - this.timeCurrentPageCreated_) > TIMEOUT_LIMIT;
   }
 
+  /**
+   * Users may put an 'next-page-no-ad' attribute on their pages to prevent ads
+   * from showing as the next page.
+   * @param {?../../amp-story/0.1/amp-story-page.AmpStoryPage} page
+   * @return {boolean}
+   * @private
+   */
+  adsAreBlocked_(page) {
+    return page.element.hasAttribute(AD_BLOCK_ATTR);
```

Comment:
update variable name and method name

Reply:
Do you mean something like `nextPageAdsAreBlocked_` and `AD_BLOCK_NEXT_PAGE`??

Reply:
Also, should we update the documentation?

Reply:
`nextPageNoAd_(page)` `NEXT_PAGE_NO_AD`

Reply:
Done. Docs coming in follow up.